data/json/items/tools.json

```diff
@@ -1098,6 +1098,52 @@
     "use_action": "HOTPLATE"
   },
   {
+    "id": "trangia_stove",
+    "sub": "hotplate",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "yellow",
+    "name": "spirit burner",
+    "description": "This is a small brass spirit burner with a screw-top lid and simmer ring, designed for cooking. It burns concentrated alcohol.",
+    "price" : 1000,
+    "material" : "copper",
+    "weight": 113,
+    "volume": 0,
```

Comment:
0 volume is not OK. Especially considering it has storage capacity of 1 volume flask.

Also, 14 max charges on a crafting item is not OK. It will only be annoying.

We don't have support for using fuel-as-charges during crafting, so all crafting tools need to have sensible charge capacity for now. If it can't be achieved with item alone, pair it with a bottle or something like that.